src/web/handlers/GestureHandler.ts

```diff
@@ -272,42 +272,60 @@ export default abstract class GestureHandler {
   // Event actions
   //
 
-  protected onPointerDown(event: AdaptedEvent): void {
+  protected onPointerDown(
+    adaptedEvent: AdaptedEvent,
+    _originalEvent?: TouchEvent | PointerEvent
```

Comment:
On Android handler methods also accept two events, and the untransformed one is called `sourceEvent`, I think we should stay consistent with names.

Reply:
Done in [5c26a86](https://github.com/software-mansion/react-native-gesture-handler/pull/2631/commits/5c26a86fd4ccfd2a17e2d54e49d54a50c5b2843a)